Tests break for PyPy-3.7 on GHA

Traceback:

```
======================================================================
ERROR: zope.interface.common.tests.test_collections (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: zope.interface.common.tests.test_collections
Traceback (most recent call last):
  File "/opt/hostedtoolcache/PyPy/3.7.12/x64/lib-python/3/unittest/loader.py", line 436, in _find_test_path
    module = self._get_module_from_name(name)
  File "/opt/hostedtoolcache/PyPy/3.7.12/x64/lib-python/3/unittest/loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "/home/runner/work/zope.interface/zope.interface/src/zope/interface/common/tests/test_collections.py", line 138, in <module>
    add_abc_interface_tests(TestVerifyClass, collections.ISet.__module__)
  File "/home/runner/work/zope.interface/zope.interface/src/zope/interface/common/tests/__init__.py", line 50, in add_abc_interface_tests
    add_verify_tests(cls, iter_abc_interfaces(predicate))
  File "/home/runner/work/zope.interface/zope.interface/src/zope/interface/common/tests/__init__.py", line 70, in add_verify_tests
    assert not hasattr(cls, name), (name, list(cls.__dict__))
AssertionError: ('test_auto_ParseResults_zope_interface_common_collections_IMutableMapping', ['__module__', 'test_frozenset', 'test_list', 'test_UserList', 'test_UserDict', 'test_UserString', 'test_non_iterable_UserDict', 'UNVERIFIABLE', 'UNVERIFIABLE_RO', '__doc__', 'maxDiff', 'test_auto_list_zope_interface_common_collections_IIterable', 'test_auto_ro_list_zope_interface_common_collections_IIterable', 'test_auto_sequenceiterator_zope_interface_common_collections_IIterator', 'test_auto_ro_sequenceiterator_zope_interface_common_collections_IIterator', 'test_auto_dict_keyiterator_zope_interface_common_collections_IIterator', 'test_auto_ro_dict_keyiterator_zope_interface_common_collections_IIterator', 'test_auto_dict_valueiterator_zope_interface_common_collections_IIterator', 'test_auto_ro_dict_valueiterator_zope_interface_common_collections_IIterator', 'test_auto_dict_itemiterator_zope_interface_common_collections_IIterator', 'test_auto_ro_dict_itemiterator_zope_interface_common_collections_IIterator', 'test_auto_reversesequenceiterator_zope_interface_common_collections_IIterator', 'test_auto_ro_reversesequenceiterator_zope_interface_common_collections_IIterator', 'test_auto_range_iterator_zope_interface_common_collections_IIterator', 'test_auto_ro_range_iterator_zope_interface_common_collections_IIterator', 'test_auto_setiterator_zope_interface_common_collections_IIterator', 'test_auto_ro_setiterator_zope_interface_common_collections_IIterator', 'test_auto_zip_zope_interface_common_collections_IIterator', 'test_auto_ro_zip_zope_interface_common_collections_IIterator', 'test_auto_coroutine_zope_interface_common_collections_ICoroutine', 'test_auto_ro_coroutine_zope_interface_common_collections_ICoroutine', 'test_auto_tuple_zope_interface_common_collections_ISequence', 'test_auto_ro_tuple_zope_interface_common_collections_ISequence', 'test_auto_str_zope_interface_common_collections_ISequence', 'test_auto_ro_str_zope_interface_common_collections_ISequence', 'test_auto_range_zope_interface_common_collections_ISequence', 'test_auto_ro_range_zope_interface_common_collections_ISequence', 'test_auto_memoryview_zope_interface_common_collections_ISequence', 'test_auto_ro_memoryview_zope_interface_common_collections_ISequence', 'test_auto_Row_zope_interface_common_collections_ISequence', 'test_auto_ro_Row_zope_interface_common_collections_ISequence', 'test_auto_UserString_zope_interface_common_collections_ISequence', 'test_auto_ro_UserString_zope_interface_common_collections_ISequence', 'test_auto_frozenset_zope_interface_common_collections_ISet', 'test_auto_ro_frozenset_zope_interface_common_collections_ISet', 'test_auto_mappingproxy_zope_interface_common_collections_IMapping', 'test_auto_ro_mappingproxy_zope_interface_common_collections_IMapping', 'test_auto_dict_zope_interface_common_collections_IMapping', 'test_auto_ro_dict_zope_interface_common_collections_IMapping', 'test_auto_dict_values_zope_interface_common_collections_IValuesView', 'test_auto_ro_dict_values_zope_interface_common_collections_IValuesView', 'test_auto_generator_zope_interface_common_collections_IGenerator', 'test_auto_ro_generator_zope_interface_common_collections_IGenerator', 'test_auto_dict_items_zope_interface_common_collections_IItemsView', 'test_auto_ro_dict_items_zope_interface_common_collections_IItemsView', 'test_auto_dict_keys_zope_interface_common_collections_IKeysView', 'test_auto_ro_dict_keys_zope_interface_common_collections_IKeysView', 'test_auto_async_generator_zope_interface_common_collections_IAsyncGenerator', 'test_auto_ro_async_generator_zope_interface_common_collections_IAsyncGenerator', 'test_auto_list_zope_interface_common_collections_IMutableSequence', 'test_auto_ro_list_zope_interface_common_collections_IMutableSequence', 'test_auto_bytearray_zope_interface_common_collections_IMutableSequence', 'test_auto_ro_bytearray_zope_interface_common_collections_IMutableSequence', 'test_auto_deque_zope_interface_common_collections_IMutableSequence', 'test_auto_ro_deque_zope_interface_common_collections_IMutableSequence', 'test_auto_UserList_zope_interface_common_collections_IMutableSequence', 'test_auto_ro_UserList_zope_interface_common_collections_IMutableSequence', 'test_auto_bytes_zope_interface_common_collections_IByteString', 'test_auto_ro_bytes_zope_interface_common_collections_IByteString', 'test_auto_bytearray_zope_interface_common_collections_IByteString', 'test_auto_ro_bytearray_zope_interface_common_collections_IByteString', 'test_auto_set_zope_interface_common_collections_IMutableSet', 'test_auto_ro_set_zope_interface_common_collections_IMutableSet', 'test_auto_dict_zope_interface_common_collections_IMutableMapping', 'test_auto_ro_dict_zope_interface_common_collections_IMutableMapping', 'test_auto_ParseResults_zope_interface_common_collections_IMutableMapping', 'test_auto_ro_ParseResults_zope_interface_common_collections_IMutableMapping'])


----------------------------------------------------------------------
Ran 1223 tests in 12.059s

FAILED (errors=1, skipped=14)
```

See https://github.com/zopefoundation/zope.interface/runs/5356262011?check_suite_focus=true

--- a/src/zope/interface/common/tests/__init__.py
+++ b/src/zope/interface/common/tests/__init__.py
@@ -60,7 +60,8 @@ def add_verify_tests(cls, iface_classes_iter):
 
                 self.assertTrue(self.verify(iface, stdlib_class))
 
-            suffix = "%s_%s_%s" % (
+            suffix = "%s_%s_%s_%s" % (
+                stdlib_class.__module__.replace('.', '_'),
                 stdlib_class.__name__,
                 iface.__module__.replace('.', '_'),
                 iface.__name__

